lib/widgets/auth/email_auth_options_page.dart

```diff
@@ -364,3 +309,69 @@ class CreateAccountButton extends StatelessWidget {
     );
   }
 }
+
+class SwitchModeText extends StatelessWidget {
```

Comment:
adding a text+link combo that does the same as the chips but looks less shit

Reply:
chip revolution